tests/TestModels/Client.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Spatie\Permission\Tests\TestModels;
+
+use Illuminate\Contracts\Auth\Access\Authorizable as AuthorizableContract;
+use Illuminate\Foundation\Auth\Access\Authorizable;
+use Laravel\Passport\Client as BaseClient;
+use Spatie\Permission\Traits\HasRoles;
+
+class Client extends BaseClient implements AuthorizableContract
+{
+    use HasRoles;
+    use Authorizable;
+
+    /**
+     * Required to make clear that the client requires the api guard
+     *
+     * @var string
+     */
+    protected $guard_name = 'api';
```

Comment:
FYI Another option could be the following. This package inspects for both the method or the property.
```php
    public function guardName()
    {
        return 'api'; // or perhaps a config() value;
    }
```

Reply:
It could be a config value. But I don't know if that would be required since this Model is only used for testing. And I guess, in the docs we will describe what is required to use the Passport Client. And so the guard name is up for the end user the decide? I don't think that this package should include the extended Client class (besides the one for testing)?